thunder/tests/framework.py

```diff
@@ -192,7 +191,7 @@ def executors_list(self) -> list[extend.Executor]:
         return [executors.get_nvfuser_executor()]
 
     def version(self):
-        return executors.get_nvfuser_executor().version()
+        return executors.get_nvfuser_executor().version
```

Comment:
QQ: Is this function not used at all?

Reply:
That's correct so nobody noticed the bug. At some point, I tried to version check nvFuserTestExecutor and ran into this bug. I don't think the latest snapshot uses it either.